### PR TITLE
Update ModLenses Integration (Hide Whitespace Changes required)

### PR DESCRIPTION
**Pull request by the-m4a**
I went through an updated the MoreLenses integration, including adopting the current folder structure used by ML.  The primary difference between this CQUI integration and the ML integration is in the filename casing:  ML currently has all references to filenames lowercased, however, it does not look like (at least in my Git repo) that the files are actually lowercased.

I verified this loaded and worked on Exp2.  I'm setting this as a draft request right now as I want to take time later to verify a bit more, though I think it'll work just fine, at least on Windows.  I'm hoping @RustyD or @chaorace can give this branch a go on Linux and see if the Minimap Panel and lenses will at least load.

**Note:** Git appears to have not liked the "git move" for the files.  Lame.  By and large, there's no code to specifically review since it's just integrating an already released mod.  The "CQUI Specific" sections were marked previously and remain so in this instance.

### CODE REVIEW

File: Integrations/ML/Lenses/Archaeologist/ModLens_Archaeologist.lua [hunk 1]
```diff
@@ -0,0 +1,156 @@
+local LENS_NAME = "ML_ARCHAEOLOGIST"
```
Comment (the-m4a) [inline Integrations/ML/Lenses/Archaeologist/ModLens_Archaeologist.lua:1]: this is annoying - I did the proper Git move command.  Git doesn't like the move + changes, in this case, it seems?

image

**Comment by Infixo** [comment]
Can we please wait for a proper access to lua.log from Linux? Without it this is going to be a blind chase.

**Comment by the-m4a** [comment]
is why it's a draft - just here to be available, etc.  more lenses needed updating anyway, so eventually this becomes a thing.  no rush on this one

**Comment by RustyD** [comment]
Well, I've just had another quick check, and there is no 'lua.log' in my Logs folder. 

So, unless there is a lua expert who might be able to suggest why this is missing from my installation, then it seems like my contribution is of low value? That would be a shame.

**Comment by Infixo** [comment]
On Win, there are 2 files with config options, AppOptions.txt and UserOptions.txt. There are many logging and debug options in them. If you find them, then pls. post and we'll see if nothing is missing in them.

File: CQUI.modinfo [hunk 2]
```diff
@@ -836,13 +836,13 @@
...
             </Properties>
             <Items>
-                <File>Integrations/ML/Lenses/ModLens_Builder.lua</File>
-                <File>Integrations/ML/Lenses/ModLens_Scout.lua</File>
-                <File>Integrations/ML/Lenses/ModLens_Archaeologist.lua</File>
-                <File>Integrations/ML/Lenses/ModLens_Barbarian.lua</File>
-                <File>Integrations/ML/Lenses/ModLens_Naturalist.lua</File>
-                <File>Integrations/ML/Lenses/ModLens_Wonder.lua</File>
-                <File>Integrations/ML/Lenses/ModLens_CitizenManagement.lua</File>
+                <File>Integrations/ML/Lenses/Archaeologist/ModLens_Archaeologist.lua</File>
```
Comment (Infixo) [inline CQUI.modinfo:839]: Why is each lens in a separate folder? In my ML install from Steam all lenses are directly in the Lens folder, no sub-folders.

**Comment by Infixo** [comment]
The "rework folder structure" commit in ML's github is from 1 Jun. The latest release on Steam is from 23 May.
So, the new folder structure has not yet been released.

**Comment by RustyD** [comment]
> On Win, there are 2 files with config options, AppOptions.txt and UserOptions.txt. There are many logging and debug options in them. If you find them, then pls. post and we'll see if nothing is missing in them.

Ok, attached. I've tried turning on some of the 'debug options' - not sure if that has made a difference.


[Options.zip](https://github.com/civfanatics/CQUI_Community-Edition/files/5214180/Options.zip)

**Comment by Infixo** [comment]
AppOptions seems ok. In UserOptions, set to 1 all "AI...Log" params and GameEffectsLogLevel = 2. Also you may try ProfileEnabled  = 1, I don't remember if I have it on 1 or 0. Same with GameHistoryLogLevel - not sure what it does.
Imho, lua.log is like a default file. I think it should on everyone's system by default. So idk, check again that Civ6 can write into that folder, etc. Also, I'd like other Linux guys to help with that issue.

**Comment by RustyD** [comment]
> AppOptions seems ok. In UserOptions, set to 1 all "AI...Log" params and GameEffectsLogLevel = 2. Also you may try ProfileEnabled = 1, I don't remember if I have it on 1 or 0. Same with GameHistoryLogLevel - not sure what it does.
> Imho, lua.log is like a default file. I think it should on everyone's system by default. So idk, check again that Civ6 can write into that folder, etc. Also, I'd like other Linux guys to help with that issue.

I tried all those settings you suggested, and also made the 'Logs' folder writable by all. Still no joy. It must be something else about my system somewhere.

Actually, thinking about it, I may be able to try it out on my old desktop system and see if I get the same issue there. I'll probably need a day or two to get around to that, but maybe I'll have more joy there.

**Comment by the-m4a** [comment]
I am closing this for now, will re-do the integration update at some point in the future.

**Pull request closed without merging**